docs/tutorial/get-started.md

```diff
@@ -0,0 +1,239 @@
++---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| StoragePool          | Node           | Driver    | PoolName                         | FreeCapacity | TotalCapacity | CanSnapshots | State | SharedName |
+|=========================================================================================================================================================|
+| DfltDisklessStorPool | n1.example.com | DISKLESS  |                                  |              |               | False        | Ok    |            |
+| DfltDisklessStorPool | n2.example.com | DISKLESS  |                                  |              |               | False        | Ok    |            |
+| DfltDisklessStorPool | n3.example.com | DISKLESS  |                                  |              |               | False        | Ok    |            |
+| pool1                | n1.example.com | FILE_THIN | /var/lib/piraeus-datastore/pool1 |    24.54 GiB |     49.30 GiB | True         | Ok    |            |
+| pool1                | n2.example.com | FILE_THIN | /var/lib/piraeus-datastore/pool1 |    23.03 GiB |     49.30 GiB | True         | Ok    |            |
+| pool1                | n3.example.com | FILE_THIN | /var/lib/piraeus-datastore/pool1 |    26.54 GiB |     49.30 GiB | True         | Ok    |            |
++---------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+## 4: Using Piraeus Datastore
+
+We now have successfully deployed and configured Piraeus Datastore, and are ready to create our first
+[`PersistentVolume`](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) in Kubernetes.
+
+First, we will set up a new [`StorageClass`](https://kubernetes.io/docs/concepts/storage/storage-classes/) for our
+volumes. In the `StorageClass`, we specify the storage pool from above:
+
+```
+$ cat <<EOF | kubectl apply -f -
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: piraeus-storage
+provisioner: linstor.csi.linbit.com
+allowVolumeExpansion: true
+volumeBindingMode: WaitForFirstConsumer
+parameters:
+  linstor.csi.linbit.com/storagePool: pool1
+EOF
+```
+
+Next, we will create a [`PersistentVolumeClaim`](https://kubernetes.io/docs/concepts/storage/persistent-volumes/),
+requesting 1G of storage from our newly created `StorageClass`.
+
+```
+$ cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: data-volume
+spec:
+  storageClassName: piraeus-storage
+  resources:
+    requests:
+      storage: 1G
+  accessModes:
+    - ReadWriteOnce
+EOF
+```
+
+When we check the created PersistentVolumeClaim, we can see that it remains in `Pending` state. We first need to create
+a "consumer", in this case is just a `Pod`.
```

Comment:
```suggestion
a "consumer", which in this case is just a `Pod`.
```

Maybe move this sentence below the `kubectl get persistentvolumeclaim`. That is connected more to the previous sentence.